musync_save/hit_delay.py

```
@@ -660,7 +660,11 @@ def _action_get_console_data(self) -> None:
             return
 
         # 3. 统计计算
-        avg_delay: float = sum(hit_delays) / len(hit_delays) if hit_delays else 0.0
+        # NarrowDelayInterval 为 True 时用 45ms，False 时用 90ms
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)